packages/block-library/src/embed/index.js

```diff
@@ -142,14 +161,15 @@ export function getEmbedEdit( title, icon ) {
 		 * if the HTML has an iframe with width and height set.
 		 *
 		 * @param {string} html The preview HTML that possibly contains an iframe with width and height set.
+		 * @return {string} The CSS classes with any aspect ratio classes included.
 		 */
-		maybeSetAspectRatioClassName( html ) {
+		getClassNamesWithAspectRatio( html ) {
 			const previewDocument = document.implementation.createHTMLDocument( '' );
 			previewDocument.body.innerHTML = html;
 			const iframe = previewDocument.body.querySelector( 'iframe' );
 
 			if ( ! iframe ) {
-				return;
+				return this.props.attributes.className;
```

Comment:
This function could be refactored a bit to avoid so many returns. Could remove this if statement, change the one below to `if( iframe && iframe.height && iframe.width )`, and then move the remaining `return this.props.attributes.className;` to the very last line of the function.

Reply:
Gah yes, I missed a refactor pass!